client/src/components/MultiJurisdictionAudit/RoundManagement/index.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { render, screen, fireEvent } from '@testing-library/react'
+import { useParams } from 'react-router-dom'
+import RoundManagement from './index'
+import { roundMocks, auditBoardMocks } from '../_mocks'
+import { dummyBallots } from '../../SingleJurisdictionAudit/_mocks'
+import { withMockFetch } from '../../testUtilities'
+import {
+  downloadPlaceholders,
+  downloadLabels,
+  downloadDataEntry,
+} from './generateSheets'
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
+  useRouteMatch: jest.fn(),
+  useParams: jest.fn(),
+}))
+const paramsMock = useParams as jest.Mock
+paramsMock.mockReturnValue({
+  electionId: '1',
+  jurisdictionId: '1',
+})
+
+jest.mock('./generateSheets', () => ({
+  downloadPlaceholders: jest.fn(),
+  downloadLabels: jest.fn(),
+  downloadDataEntry: jest.fn(),
+}))
+const downloadPlaceholdersMock = downloadPlaceholders as jest.Mock
+const downloadLabelsMock = downloadLabels as jest.Mock
+const downloadDataEntryMock = downloadDataEntry as jest.Mock
+
+window.open = jest.fn()
+
+const apiCalls = {
+  getBallots: {
+    url: '/api/election/1/jurisdiction/1/round/round-1/ballots',
+    response: dummyBallots,
+  },
+}
+
+describe('RoundManagement', () => {
```

Comment:
do we need a test for the flow where the JA creates the audit boards?

Reply:
@jonahkagan That is tested in the CreateAuditBoards component directly.